internal/test/consensus/scenarios/fork_and_select_v1/configurator.sh

```
@@ -0,0 +1,331 @@
+
+# Paths inside the configurator container.
+STAGING_ROOT=/staging
+LOG_ROOT=/var/log
+SHARED_PREFIX_DIR=/configs/shared-prefix-db
+PEER_A_DATA_DIR=/peer-a-data
+PEER_B_DATA_DIR=/peer-b-data
+
+# ─── helpers ────────────────────────────────────────────────────────
+
+write_file() {
+    local tmp_file="${1}_$(tr </dev/urandom -dc A-Za-z0-9 | head -c16)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)